### PR TITLE
Build 0 for portcullis 1.2.0

### PR DESCRIPTION
**Pull request by lucventurini**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by npavlovikj** [comment]
Try setting the flags for `zlib` errors, https://bioconda.github.io/troubleshooting.html#zlib-errors.

**Comment by lucventurini** [comment]
Thanks, I will try tomorrow!

**Comment by maplesond** [comment]
I had a look at this, and reenabled the build environment flags as it appears these are required for zlib.  Still encountering the same boost issue though.  Are there any docs for using bioconda boost?  Build and runtime linking for boost works fine on most systems with a version of boost installed from a package manager.

**Comment by lucventurini** [comment]
Dear @npavlovikj , any insight on this?

**Comment by npavlovikj** [comment]
@lucventurini , can you please try adding these two lines in "build.sh":
```
export LIBS="-lm ${PREFIX}/lib/libz.a"
 ./configure --disable-silent-rules --disable-dependency-tracking --prefix=$PREFIX --with-boost=${PREFIX} --with-boost-libdir=${PREFIX}/lib
```
?

**Comment by npavlovikj** [comment]
Also, `boost` needs to be added to the `run` requirements.

**Comment by lucventurini** [comment]
Hi @npavlovikj, thank you for the suggestions, much appreciated! Test currently underway.

**Comment by lucventurini** [comment]
Dear @npavlovikj , it seems to have worked! If you could please authorize the merging, we would ever be so grateful.

File: recipes/portcullis/meta.yaml [hunk 1]
```diff
@@ -28,14 +28,18 @@ requirements:
     - automake >=1.15.1
     - libtool
     - perl
+    - boost
```
Comment (bgruening) [inline recipes/portcullis/meta.yaml]: I don't think boost and zlib should be in build. host should be enough

Comment (lucventurini) [reply recipes/portcullis/meta.yaml]: Updated according to the suggestion. Waiting for the build to complete.

**Comment by lucventurini** [comment]
Hi @bgruening , I have updated the recipe according to your suggestion (ie removing boost and zlib from the "build" section). Everything seems to be working, if so, we would be grateful if you could authorise the merge! 
Kind regards

File: recipes/portcullis/meta.yaml [hunk 2]
```diff
@@ -33,9 +33,10 @@ requirements:
     - pandas
     - python
     - samtools >=1.9
+    - boost
   run:
     - python
-    #- zlib
+    - boost
```
Comment (epruesse) [inline recipes/portcullis/meta.yaml]: I think you want `boost-cpp`, not `boost`.

Comment (lucventurini) [reply recipes/portcullis/meta.yaml]: Testing now with boost-CPP. If it passes, would it be possible to merge into master, please?

**Comment by lucventurini** [comment]
@epruesse yes please re: being added as the maintainer. Thank you for taking the time to review this recipe!

**Comment by epruesse** [comment]
@lucventurini To do that, just add the fragment I posted above to the meta.yaml next time you update the recipe.

**Pull request merged**